benchmarks/templates/benchmarks/participate.html

```diff
@@ -23,6 +23,7 @@ <h4 class="title is-4">Change the evaluation: Submit a metric</h4>
             </div>
 
             <h4 class="title is-4">Citation</h4>
+    <div class="block">
           {% if "language" in request.path %}
                <h5>Language citation needed...</h5>
           {% else %}
```

Comment:
Unify with the code below:
```suggestion
                If you use Brain-Score in your work, please cite
                <a href="{{ citation_domain_url }}">
                    <em>{{ citation_domain_title }}</em>
                </a> (technical)
                and
                <a href="{{ citation_general_url }}">
                    <em>{{ citation_general_title }}</em>
                </a> (perspective)
                as well as the respective benchmark sources.
                <pre>
                    {{ citation_domain_bibtex }}
                    
                    {{ citation_general_bibtex }}
                </pre>
```

For the context, you would set
```
{
  'citation_general_url': 'https://www.cell.com/neuron/fulltext/S0896-6273(20)30605-X',
  'citation_general_title': 'Integrative Benchmarking to Advance Neurally Mechanistic Models of Human Intelligence',
  'citation_general_bibtex': '@article{Schrimpf2020integrative, ...',
}
```
And set the `citation_domain_*` to the current 2018 paper for vision. For language, I would say put https://www.pnas.org/content/118/45/e2105646118 for now, but we should maybe replace that with a to-be-written technical whitepaper at some point.